README.md

```diff
@@ -437,6 +477,7 @@ The built-in reporter options are:
 - sarif
 - sonar (SonarQube generic issue format)
 - unix
+- tsc (compatible to TypeScript compiler)
```

Comment:
Hi, 

sorry for coming back to you so late, but I had some severe health issues.

You can find the formatter for typescript compiler [here](https://github.com/microsoft/TypeScript/blob/d4fbc9b57d9aa7d02faac9b1e9bb7b37c687f6e9/src/compiler/program.ts#L663C1-L674C2),

A typical use case is for me chaining the compilation by using [scripts](https://docs.npmjs.com/cli/v10/using-npm/scripts).

prepare -> prePrepare -> compileTs -> preCompileTs -> protoc -> preProtoc -> protolint

When running `npm prepare`, this will run compileTs als prePrepare hook.
When running compileTs, it will run the protoc as preCompileTs hook.
When running protoc, it will run protolint as preProtoc hook.

When using your IDE, you can then use the problem matcher for TSC to get the issues from protolint.

I can remove the matching commits, if you this does not fit in your plans.

Reply:
Hello,

No problem at all. I hope you are feeling better now. 
Thank you for your clear explanation; it's quite understandable.